app/shared/actions/accounts.js

```diff
@@ -92,29 +92,33 @@ export function getAccount(account = '') {
   };
 }
 
-export function getActions(account = '') {
+export function getAccountActions(amount) {
```

Comment:
The UI should be passing the account name, it shouldn't automatically assume you're using the `settings.account` variable. 

It also shouldn't have been renamed - if you needed a new function, write a new function. The same goes with all of the actions constants modified below.

In order to support multiple accounts this function needs to support calls for more than one account.

Reply:
I just thought that getAccountAction was a more relevant name since it returns the actions associated to an account. Will change it though.

Reply:
The point was that if you needed a function that called it, it'd be better to just write a second function rather than rewriting the existing one to do what you wanted it to. 

What existed was:

```
function getActions(account) {}
``` 
and instead of rewriting it, it would have been easier to just add an additional function that called the first one.

```
function getAccountActions(amount) {
   const account = settings.account;
   getActions(account, amount)
}
```